Database/DB-week1/Homework.sql

```diff
@@ -0,0 +1,115 @@
+-- 1. Find out how many tasks are in the task table
+SELECT COUNT(*) AS total_tasks
+FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*)
+FROM task
+WHERE due_date IS NULL;
+
+-- 3. Find all the tasks that are marked as done
+SELECT *
+FROM task
+WHERE status_id = 3;
+
+-- 4. Find all the tasks that are not marked as done
+SELECT *
+FROM task
+WHERE status_id != 3;
+
+-- 5. Get all the tasks, sorted with the most recently created first
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+-- 6. Get the single most recently created task
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+-- 7. Get the title and due date of all tasks where the title or description contains database
+SELECT title,
+       due_date
+FROM task
+WHERE title LIKE '%database%'
+   OR description LIKE '%database%';
+
+-- 8. Get the title and status (as text) of all tasks
```

Comment:
this query is wrong, please take a second look

Reply:
the query works just fine and I can see the results containing 'database', could you be more specific what exactly is wrong in this query?
<img width="930" alt="image" src="https://github.com/user-attachments/assets/0ca07313-9f06-4b25-a6b7-6c6da0826e1d">